storage/buckets/sharing/index.md

```diff
@@ -7,22 +7,26 @@ permalink: /storage/buckets/sharing/
 {:toc}
 
 Bucket sharing allows you to share data between projects within an [organization](/management/organization/),
-giving you far greater organizational control over your data. 
-You can now decide which projects, and therefore which users, have access to which data. 
-This also helps speed up your data workflow and reduce your project usage totals because the data size 
+giving you far greater organizational control over your data.
+You can now decide which projects, and therefore which users, have access to which data.
+This also helps speed up your data workflow and reduce your project usage totals because the data size
 and rows are counted only in the source project.
 
 - **Shared bucket** --- source project bucket with **Sharing** enabled; it contains data to be available in other projects.
 - **Linked bucket** --- reference to a shared bucket in any of the destination projects.
 
-All changes and events in a shared bucket are propagated to its linked buckets; 
+All changes and events in a shared bucket are propagated to its linked buckets;
 any tables you create in the shared bucket become immediately available in the destination projects.
 
 ## Sharing Type
 You can specify who can link the bucket to a project:
 
-- **Organization Member** --- Only organization members are able to link the shared bucket to a project.
-- **Project Member** --- Every project member in the organisation is able to link the shared bucket to a project.
+- **Project Member** --- To be able to link the bucket to a project, a person must be a user of the destination project.
+- **Organization Member** --- To be able to link the bucket to a project, a person must be a user of the destination project
+*and also a member of that organization*.
```

Comment:
This wording is still confusing to me.  To me from a language POV, all project members would be members of the organisation.  I wish we could change it to role instead.  Something like `*and also be an administrator of that organization *`

Reply:
understand, but i'd like to honor the UI naming
![image](https://user-images.githubusercontent.com/4319320/46654458-2c6c3280-cba9-11e8-9079-191bb9470a7a.png)

which unfortunately is also chaos https://github.com/keboola/kbc-ui/issues/2127

Reply:
I see, ok I guess this is fine and we can update it when the UI changes